synapse/storage/databases/main/event_federation.py

```diff
@@ -71,6 +72,12 @@
 
 logger = logging.getLogger(__name__)
 
+BACKFILL_EVENT_BACKOFF_UPPER_BOUND_SECONDS: int = int(
+    datetime.timedelta(days=7).total_seconds()
+)
+BACKFILL_EVENT_EXPONENTIAL_BACKOFF_STEP_SECONDS: int = int(
+    datetime.timedelta(hours=1).total_seconds()
+)
```

Comment:
Any opinions on these values?

I chose 7 days because trying again next week seemed reasonable if someone's server was offline and then they got it working again.

This means it takes 8 attempts to get to the upper bound (`2^7 = 128`, `2^8 = 256`)

2hr, 4hr, 8hr, 16hr, 32hr, 64hr, 128hr, (capped at 168hr from now on)